CONTRIBUTION.md

```diff
@@ -28,58 +28,126 @@ We know that as of 1.12 this is not the case for great parts of the mod, but we
 Unless your PR is directly related to code cleaning then your PR should not contain any formatting changes to unchanged code blocks as this simply makes reviewing them unnecessarily complex.
 
 ### Formatting
-* Use 4 space indent instead of Tabs
-* operators should be surrounded by spaces
-  * For example `if (statement == true)`
-* a `,` comma to separate types should be followed by exactly one space
-* type casts should be followed by exactly one space
-* `{` should not be prefixed with returns
-* Methods should have exactly one free line between them (or their doc blocks)
-* Avoid empty lines at the start of class, interface or enum definitions
-* Avoid unnecessary lines in empty Methods
-    * `public void emptyImpl() {}`
-    * Also try and avoid introducing code that may require empty methods
-* Method/Class annotations should gain their own lines in front of the definition
-* Parameter Annotations should be surrounded by exactly one space
-* Comments should always start with a single space `// Comment`
+* Use 4 space indent instead of Tabs.
+* Operators should be surrounded by spaces:
+  * For example `if (statement == true)`.
+* A `,` comma to separate types should be followed by exactly one space.
+* Type casts should be followed by exactly one space.
+* `{` should not be wrapped onto a new line.
+* Methods should have exactly one free line between them (or their doc blocks).
+* Avoid empty lines at the start of class, interface or enum definitions.
+* Avoid unnecessary lines in empty Methods:
+  * `public void emptyImpl() {}`.
+  * Try to avoid introducing code that may require empty methods.
+* Method/Class annotations should gain their own lines in front of the definition.
+* Parameter Annotations should be surrounded by exactly one space.
+* Comments should always start with a single space `// Comment`.
+* Import statments should be ordered in sections with exactly 1 empty line between them. Regular imports go in the front and static imports follow them.
+  * If you are using IntelliJ, this is the default order.
+  * If you are using Eclipse, you need to reorder them under `Preferences -> Java -> Code Style -> Organize Imports`.
```

Comment:
No to all of this. There is no point to ordering imports.

Reply:
Note that by default, eclipse puts `java.*` in the front and IntelliJ puts them in the back. Even though we have the config already, we should the order clear.

Reply:
But why? You are only making more of an argument to not enforce this.

Reply:
We were having the problem with people using Eclipse has their (`java.*`) imports on the top, and then auto formatting of IDEA puts them in the bottom.
This creates a bunch of useless commit history where imports statements were being moved around constantly.